src/operations.rs

```diff
@@ -156,6 +157,7 @@ pub(crate) fn aesdec(value: u128, xor: u128) -> u128 {
     use core::arch::aarch64::*;
     #[cfg(target_arch = "arm")]
     use core::arch::arm::*;
+    use core::mem::transmute;
```

Comment:
If you use replace the remaining `transmute`s with `transmute!`, you don't need this, and can get rid of more `unsafe` code.